input/cql/AppropriateDXAScansForWomenUnder65FHIR.cql

```diff
@@ -121,24 +122,27 @@ define "First BMI in Measurement Period Less Than or Equal to 20 kg m2":
     where FirstBMI.value as Quantity <= 20 'kg/m2'
 
 define "First BMI in Measurement Period":
-  First(((["observation-bmi"]).isObservationBMI()) BMIRatio
+  First(((["USCoreBMIProfile"]).isObservationBMI()) BMIRatio
       where BMIRatio.effective.toInterval() during "Measurement Period"
         and BMIRatio.value as Quantity is not null
       sort by start of effective.toInterval()
   )
 
 define "First Average Number of Drinks Assessments Indicating More Than Two Per Day":
-  First((([Observation: "Alcoholic drinks per day"]).isAssessmentPerformed()) AverageDrinks
+  First((([ObservationScreeningAssessment: "Alcoholic drinks per day"]).isAssessmentPerformed()) AverageDrinks
       where start of AverageDrinks.effective.toInterval() during "Measurement Period"
         and AverageDrinks.value as Quantity > 2 '{drinks}/d'
       sort by start of effective.toInterval()
   )
 
+/*
+@QICore6Uplift: Per Authoring Patterns, "prevalence period would only be expected to be present on problem list items and health concerns”
+*/
 define "Has Risk Factor Any Time in History Prior to Measurement Period":
-  exists ( ( [Condition: "Osteoporosis Without Current Fracture"] OsteoporosisDiagnosis
+  exists ( ( [ConditionProblemsHealthConcerns: "Osteoporosis Without Current Fracture"] OsteoporosisDiagnosis
```

Comment:
Should be the union of the Conditions

Reply:
Based on the Authoring Patterns page (https://github.com/cqframework/CQL-Formatting-and-Usage-Wiki/wiki/Authoring-Patterns-QICore-v6.0.0), it seems that the presence of prevalenceInterval is associated with ConditionProblemsHealthConcerns.

"Problem list items and health concerns are typically documented with additional information about the condition such as prevalence period and clinical status... In particular, clinical status and prevalence period would only be expected to be present on problem list items and health concerns." Am I interpreting this correctly?